source/numeric/SVInt.cpp

```diff
@@ -485,15 +485,24 @@ void SVInt::shrinkToFit() {
         *this = resize(minBits);
 }
 
+bitwidth_t SVInt::unsignedAmount() const {
+    // [11.4.10] The right operand of shift operators is always treated as an unsigned number
+    bitwidth_t bits = getActiveBits();
+    if (bits > sizeof(bitwidth_t) * CHAR_BIT)
+        return std::numeric_limits<bitwidth_t>::max();
+    return static_cast<bitwidth_t>(isSingleWord() ? val : pVal[0]);
```

Comment:
You can replace the conditional here with just *getRawPtr()

Reply:
Yes, I will change it.